theano/tensor/tests/test_opt.py

```diff
@@ -2922,7 +2924,7 @@ def test_local_IncSubtensor_serialize():
     # Now test that the stack trace is copied over properly,
     # if we return the gradients. We need to use same mode as before.
     f = theano.function([i, j, t], dW, mode=mode)
-    assert check_stack_trace(f, ops_to_check='all')
+    assert check_stack_trace(f, ops_to_check=tensor.AdvancedIncSubtensor)
```

Comment:
We would want to include IncSubtensor and AdvancedIncSubtensor1 in there as well, I think. For instance:
`ops_to_check=[tensor.Subtensor, tensor.AvancedIncSubtensor1, tensor.AdvancedIncSubtensor]`.
If that is not working, please leave a note, as it may indicate a problem to investigate.